static/src/javascripts-legacy/projects/common/modules/experiments/tests/sleeve-notes-email-variants.js

```diff
@@ -0,0 +1,80 @@
+define([
+    'bean',
+    'fastdom',
+    'qwery',
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/fastdom-promise'
+], function (
+    bean,
+    fastdom,
+    qwery,
+    $,
+    config,
+    fastdomPromise
+) {
+    return function () {
+        this.id = 'SleevenotesEmailVariants';
+        this.start = '2017-03-03';
+        this.expiry = '2017-03-31';
+        this.author = 'Leigh-Anne Mathieson';
+        this.description = 'Using the wonderful frontend AB testing framework to AB test emails, since the AB ' +
+            'function in ExactTarget re-randomises all recipients on each send, and we need users to receive their ' +
+            'variant for several weeks. This test will ensure users are added to the corresponding email list ' +
+            '(listId) in ExactTarget';
+        this.audience = 0.6;
```

Comment:
I would start of with an audience size of 0. Then you can deploy live, opt into the test and manually test in production, then deploy a change setting the audience size to the correct amount